src/Toolkit/Xml.php

```diff
@@ -66,13 +64,13 @@ class Xml
 	/**
 	 * Generates a single attribute or a list of attributes
 	 *
-	 * @param string|array $name String: A single attribute with that name will be generated.
-	 *                           Key-value array: A list of attributes will be generated. Don't pass a second argument in that case.
+	 * @param string|array|null $name String: A single attribute with that name will be generated.
+	 *                                Key-value array: A list of attributes will be generated. Don't pass a second argument in that case.
 	 * @param mixed $value If used with a `$name` string, pass the value of the attribute here.
 	 *                     If used with a `$name` array, this can be set to `false` to disable attribute sorting.
 	 * @return string|null The generated XML attributes string
 	 */
-	public static function attr($name, $value = null): string|null
+	public static function attr(string|array|null $name, $value = null): string|null
```

Comment:
```suggestion
	public static function attr(string|array|null $name, mixed $value = null): string|null
```

Reply:
We decided not to add the mixed type hint anymore as it doesn't add any real benefit while still making PHP complain e.g. when a child class doesn't declare it.

Reply:
Got it! Then if there is no type it means mixed type.

Reply:
What if we add mixed to subclasses? Especially useful in IDE.

Reply:
We don't always have control of subclasses (plugins etc). And what would be the benefit for IDEs? Mixed means as much as "we don't know which type"

Reply:
When calling methods in the IDE, it's very helpful to see what type of argument they want and their return type. I don't fully understand why we don't want to use the `mixed` type that came officially in PHP 8.0. It's not a big deal though 👍

Reply:
But `mixed` isn't any type - so the IDE doesn't show you any useful info. `mixed` says "we don't know".

And PHP doesn't like it a parent class adds `Mixed` but a subclass doesn't. We accept that risk for when type hinting adds benefit - but in the case of mixed it doesn't add any benefit, so we said we'd rather leave it.

Reply:
Understand, it's "pseudo" type.